### PR TITLE
Adapted to KafkaMessageConsumer  changes in eventuate-messaging-kafka. Adapted cdc processor to unicode in mysql.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: docker-compose-mariadb.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 mariadb:
-  image: eventuateio/eventuate-mariadb:0.9.0.RC3
+  image: test-eventuate-mariadb
```
Comment (cer) [inline docker-compose-mariadb.yml:2]: I think it would make sense for builds on 'master' to publish BUILD-SNAPSHOT images (as well as libraries)

Comment (dartartem) [reply docker-compose-mariadb.yml:2]: Should I implement it?

Comment (cer) [reply docker-compose-mariadb.yml:2]: Yes. https://github.com/eventuate-foundation/eventuate-common/issues/33

Comment (cer) [reply docker-compose-mariadb.yml:2]: Also, please confirm that the code changes work with the current image versions, e.g. eventuateio/eventuate-mariadb:0.9.0.RC3

**Comment by cer** [comment]
Why is an image update *required*?

**Comment by dartartem** [comment]
@cer, Hm, it seems is not required. As I wrote yesterday, tests passed with old Images. I will remove update and change name of the PR.

**Comment by cer** [comment]
Also, AFAIK Kafka had not changed so the title of PR is misleading

**Comment by dartartem** [comment]
@cer, Chris, it is changed: https://github.com/eventuate-foundation/eventuate-cdc/pull/50/commits/af6765da2ab2bb2714cd9761829fcf527006edbc

**Comment by cer** [comment]
That's not Kafka, which is a  message broker

**Comment by dartartem** [comment]
@cer, Chris, yes, I changed the name of the PR.

File: eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/BinlogEntry.java [hunk 2]
```diff
@@ -2,7 +2,23 @@
 
 import io.eventuate.common.eventuate.local.BinlogFileOffset;
 
+import java.nio.charset.StandardCharsets;
+
 public interface BinlogEntry {
   Object getColumn(String name);
   BinlogFileOffset getBinlogFileOffset();
+
+  default String convertColumnToString(String name) {
```
Comment (cer) [inline eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/BinlogEntry.java:11]: I think a better name would be `getStringColumn()`

Comment (dartartem) [reply eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/BinlogEntry.java:11]: I will change

**Comment by cer** [comment]
@dartartem CircleCI build is failing: 

```
Could not resolve all files for configuration ':eventuate-cdc-data-producer-wrappers:compileClasspath'.
> Could not find io.eventuate.messaging.kafka:eventuate-messaging-kafka-spring-producer:0.10.0.BUILD-SNAPSHOT.
```

**Pull request merged**